Index: 085/org/Do.Platform/src/Do.Platform/Do.Platform.Common/AbstractKeyBindingService.cs
===================================================================
--- 085.orig/org/Do.Platform/src/Do.Platform/Do.Platform.Common/AbstractKeyBindingService.cs
+++ 085/org/Do.Platform/src/Do.Platform/Do.Platform.Common/AbstractKeyBindingService.cs
@@ -66,7 +66,7 @@ namespace Do.Platform.Common
 			//add the event to the list of bindings
 			Bindings.Add (binding);
 			//set the bound keystring in the prefs
-			prefs.Set (binding.Description.Replace (' ', '_'), binding.KeyString);
+			prefs.Set (binding.PreferenceName.Replace (' ', '_'), binding.KeyString);
 
 			return true;
 		}
@@ -86,10 +86,10 @@ namespace Do.Platform.Common
 			}
 
 			//set the new keystring
-			Bindings.First (k => k.Description == binding.Description).KeyString = newKeyString;
+			Bindings.First (k => k.PreferenceName == binding.PreferenceName).KeyString = newKeyString;
 			
 			//save the new value in the prefs
-			prefs.Set (binding.Description.Replace (' ', '_'), binding.KeyString);
+			prefs.Set (binding.PreferenceName.Replace (' ', '_'), binding.KeyString);
 
 			if (!string.IsNullOrEmpty (binding.KeyString))
 			    Log<AbstractKeyBindingService>.Debug ("\"{0}\" now mapped to \"{1}\"", binding.Description, binding.KeyString);
